packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -86,7 +86,11 @@ const AdaptedVirtualSelect = reduxFormFieldAdapter({
 
 const serviceFormSelector = formValueSelector('serviceForm');
 const oneHourInMilliSeconds = 3600000;
+const fiveMinutesInMilliSeconds = 300000;
```

Comment:
Why not do `oneMinuteInMilliseconds` then values would be:
- `5 * oneMinuteInMilliseconds`
- `15 * oneMinuteInMilliseconds`
- `30 * oneMinuteInMilliseconds`?

Reply:
👍🏽